lib/slack-ruby-bot-server/models/team_mongo.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'common_methods.rb'
```

Comment:
I think it should be in `models/mongo/team.rb` and be called Mongo::Team, then something would alias that to just Team.